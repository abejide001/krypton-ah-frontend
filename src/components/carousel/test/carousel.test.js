import React from 'react';
import { shallow } from 'enzyme';
import Carousel from '../index';

describe('carousel conatiner test', () => {
  it('should test for the carousel component', () => {
    const wrapper = shallow(<Carousel />);
    expect(wrapper.find('Grid')).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
});
