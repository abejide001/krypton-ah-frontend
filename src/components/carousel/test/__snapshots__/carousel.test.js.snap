// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`carousel conatiner test should test for the carousel component 1`] = `
<Carousel
  autoPlay={true}
  axis="horizontal"
  centerMode={false}
  centerSlidePercentage={80}
  dynamicHeight={false}
  emulateTouch={false}
  infiniteLoop={true}
  interval={3000}
  onChange={[Function]}
  onClickItem={[Function]}
  onClickThumb={[Function]}
  selectedItem={0}
  showArrows={true}
  showIndicators={true}
  showStatus={false}
  showThumbs={false}
  statusFormatter={[Function]}
  stopOnHover={true}
  swipeScrollTolerance={5}
  swipeable={true}
  transitionTime={350}
  useKeyboardArrows={false}
  verticalSwipe="standard"
  width="100%"
>
  <div>
    <Image
      as="img"
      size="massive"
      src="https://res.cloudinary.com/ah-krypton/image/upload/v1550007680/MainImage.png"
      ui={true}
    />
    <p
      className="legend text"
    >
      <p
        className="title"
      >
        Placeholder Article Title
      </p>
      <p
        className="author"
      >
        Placeholder author name
      </p>
      <Icons
        numberofcomments={23}
        ratingIcon="rating-class"
      />
    </p>
  </div>
</Carousel>
`;
